user_guides/Template/src/main/jbake/content/rules.adoc

```diff
@@ -293,142 +293,71 @@ include::rules.inc[]
 
-All tests found to be invalid will either be placed on the Exclude List
-for that version of the {TechnologyShortName} TCK or have an alternate
-test made available.
+2.3.1 Valid Challenges
+^^^^^^^^^^^^^^^^^^^^^^
+Any test case (e.g., test class, @Test method), test case configuration (e.g., deployment descriptor), test beans, annotations, and other resources considered part of the TCK may be challenged.
+
+The following scenarios are considered in scope for test challenges:
+
+* Claims that a test assertion conflicts with the specification.
+* Claims that a test asserts requirements over and above that of the specification.
+* Claims that an assertion of the specification is not sufficiently implementable.
+* Claims that a test is not portable or depends on a particular implementation.
 
-* Tests that are placed on the Exclude List will be placed on the
-Exclude List within one business day after the determination of test
-validity. The new Exclude List will be made available to all
-{TechnologyShortName} TCK licensees on the Java Licensee Engineering website.
-* Oracle, as Maintenance Lead has the option of creating alternative
-tests to address any challenge. Alternative tests (and criteria for
-their use) will be made available on the Java Licensee Engineering
-website. +
+2.3.2 Invalid Challenges
+^^^^^^^^^^^^^^^^^^^^^^^^
+The following scenarios are considered out of scope for test challenges and will be immediately closed if filed:
 
-[NOTE]
-=======================================================================
+* Challenging an implementation’s claim of passing a test.  Certification is an honor system and these issues must be raised directly with the implementation.
+* Challenging the usefulness of a specification requirement.  The challenge process cannot be used to bypass the specification process and raise in question the need or relevance of a specification requirement.
+* Claims the TCK is inadequate or missing assertions required by the specification.  See the Improvement section, which is outside the scope of test challenges.
+* Challenges that do not represent a consensus of the implementing community will be closed until such time that the community does agree or agreement cannot be made.  The test challenge process is not the place for implementations to initiate their own internal discussions.
+* Challenges to tests that are already excluded for any reason.
+* Challenges that an excluded test should not have been excluded and should be re-added should be opened as a new enhancement request
 
-Passing an alternative test is deemed equivalent to passing the original
-test.
+Test challenges must be made in writing via the {TechnologyShortName} specification project issue tracker
+as described in link:#CJAJEAEI[Section 2.3.3, "TCK Test Appeals Steps."]
 
-=======================================================================
+All tests found to be invalid will either be placed on the Exclude List
```

Comment:
If we've decided that no alternate tests will be allowed, then "will either" should be "will".
Otherwise, need to add the "or" clause.